.nais/nais.yaml

```diff
@@ -70,6 +70,20 @@ spec:
               {{/each}}
             {{/if}}
         {{/each}}
+    outbound:
+      external:
+        - host: unleash.nais.io
```

Comment:
Kunne denne vært i vars-filene?

Reply:
den er lik for prod og dev - dont repeat yourself